zlib-ng.h

```diff
@@ -1835,6 +1835,72 @@ ZEXTERN int              ZEXPORT zng_deflateResetKeep (zng_stream *);
 ZEXTERN int ZEXPORTVA zng_gzvprintf(gzFile file, const char *format, va_list va);
 #endif
 
+typedef enum {
```

Comment:
All this should probably move up to line ~1792, above the #ifdef Z_LARGE64 parts.

Also, the comments are not following the established style for the file, they involve much more white-space than we are used to. The ones in the enum should be single-line comments for example.

Reply:
I'll change that.